### ISSUE TITLE
Temperature Values too high

### ISSUE THREAD

**Issue by misel228** [sbcshop/AiryFi_Software#3]
I've only adapted the ThingSpeak example to send the requests to my own webserver to log the data.

The measured values were consistently around 32°C and the humidity was always around 64%. Another thermometer was measuring 22 to 24°C at the same time. I assume it's because it's using too much power. The board runs really warm to the touch and it ran for only about 6h on battery.

I've been trying to adapt the deep sleep functions of the ESP but with no luck so far. Additionally, this won't shutdown the sensor itself which is also quite warm.

Can you provide an a true low power example that takes a measurement, shuts down for, say, 5 minutes and then wakes up to take another measurement and goes back to deep sleep again.

**Comment by chedim** [comment]
No, they cannot. They placed the temperature sensor right next to the battery charging circuitry that is overheating so, every time the device charges the battery it will read extremely high temps.
